src/test-helper.js

```diff
@@ -12,11 +12,6 @@ exports.initSyncFunction = initSyncFunction;
  */
 exports.initDocumentDefinitions = initDocumentDefinitions;
 
-/**
- * DEPRECATED. Use initSyncFunction instead.
- */
-exports.init = initSyncFunction;
```

Comment:
Since the test-helper module was moved to this location and the package has not been released since then, there is no good reason to keep this member around.